shell/renderer/renderer_client_base.cc

```diff
@@ -239,7 +239,7 @@ void RendererClientBase::RenderThreadStarted() {
                                                      true);
 
 #if BUILDFLAG(ENABLE_ELECTRON_EXTENSIONS)
-  extensions_client_.reset(CreateExtensionsClient());
+  extensions_client_ = CreateExtensionsClient();
```

Comment:
Could we just inline `extensions_client_ = std::make_unique<ElectronExtensionsClient>();` ? Doesn't seem like we are expecting more than a single client per process.

Reply:
Good call, that's simpler. Changed in dd1ef223416f64ae291b5dddf6891df2c1aa50dc